### PR TITLE
written the code for bubble sort recursive function

### PR DESCRIPTION
**Pull request by devu26**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#3459

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@devu26 Please provide the issue number.

**Comment by devu26** [comment]
issue number 3459

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@devu26 Rename the program to this `recursive_bubble_sort.cpp`. Update the readme file.

File: C-Plus-Plus/sort/recursive bubble sort.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	cin>>n;
+	int arr[n];
+	cout<<"enter the value of element:"<<endl;
+	for(int i=0;i<n;i++){
+		cin>>arr[i];
+	}
+	BubbleSort( arr , n);
+	Display(arr,n);
+}
+/*
```
Comment (atarax665) [inline C-Plus-Plus/sort/recursive bubble sort.cpp:38]: Add space and time complexity of your approach.

File: C-Plus-Plus/sort/recursive bubble sort.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+			if(arr[i]>arr[i+1]){
+				swap(arr[i],arr[i+1]);
+			}
+		//for recursion
+		  BubbleSort(arr,n-1);
+		}
+	}
+//function for printing array
+void Display(int arr[],int n){
+	cout<<"after sorting:"<<endl;
```
Comment (atarax665) [inline C-Plus-Plus/sort/recursive bubble sort.cpp:20]:
```suggestion
	cout<<"After sorting:"<<endl;
```

File: C-Plus-Plus/sort/recursive bubble sort.cpp [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	for(int j=0;j<n;j++){
+		cout<<arr[j]<<" ";
+	}
+}
+//program for testing above functions
+int main(){
+	int n;
+	cout<<"enter the number of element:"<<endl;
```
Comment (atarax665) [inline C-Plus-Plus/sort/recursive bubble sort.cpp:28]:
```suggestion
	cout<<"Enter the number of element:"<<endl;
```

File: C-Plus-Plus/sort/recursive bubble sort.cpp [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	cin>>n;
+	int arr[n];
+	cout<<"enter the value of element:"<<endl;
```
Comment (atarax665) [inline C-Plus-Plus/sort/recursive bubble sort.cpp:31]:
```suggestion
	cout<<"Enter the value of element:"<<endl;
```

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code using [this](https://www.techiedelight.com/tools/clike).

**Comment by atarax665** [comment]
@devu26 Are you still working on this?

**Pull request closed without merging**